index.php

```diff
@@ -0,0 +1,90 @@
+<h3>Right to restriction of processing</h3>
+<p>You have the right to request the restriction of the processing of your personal data. For this purpose, you can contact us at any time. The right to restriction of processing exists in the following cases:</p>
+<ul>
+    <li>If you dispute the accuracy of your personal data stored by us, we usually need time to verify this. For the duration of the review, you have the right to request the restriction of the processing of your personal data.</li>
+    <li>If the processing of your personal data happened/is happening unlawfully, you may request the restriction of data processing instead of erasure.</li>
+    <li>If we no longer need your personal data, but you need it to exercise, defend or enforce legal claims, you have the right to request restriction of the processing of your personal data instead of deletion.</li>
+    <li>If you have lodged an objection pursuant to Art. 21 (1) DSGVO, a balancing of your and our interests must be carried out. As long as it has not yet been determined whose interests prevail, you have the right to request the restriction of the processing of your personal data.</li>
+</ul>
+<p>If you have restricted the processing of your personal data, this data may - apart from being stored - only be processed with your consent or for the assertion, exercise or defense of legal claims or for the protection of the rights of another natural or legal person or for reasons of an important public interest of the European Union or a Member State.</p>
+<h3>SSL or TLS encryption</h3>
+<p>This site uses SSL or TLS encryption for security reasons and to protect the transmission of confidential content, such as orders or requests that you send to us as the site operator. You can recognize an encrypted connection by the fact that the address line of the browser changes from "http://" to "https://" and by the lock symbol in your browser line.</p>
+<p>If SSL or TLS encryption is activated, the data you transmit to us cannot be read by third parties.</p>
+
+<h2>3. Data collection on this website</h2>
+<h3>Server log files</h3>
+<p>The provider of the pages automatically collects and stores information in so-called server log files, which your browser automatically transmits to us. These are:</p>
+<ul>
+    <li>Browser type and version</li>
+    <li>Operating system used</li>
+    <li>Referrer URL</li>
+    <li>Host name of the accessing computer</li>
+    <li>Time of the server request</li>
+    <li>IP address</li>
+</ul>
+<p>This data is not merged with other data sources.</p>
+<p>The collection of this data is based on Art. 6 para. 1 lit. f DSGVO. The website operator has a legitimate interest in the technically error-free presentation and optimization of its website - for this purpose, the server log files must be collected.</p>
+<p>Source: <a href="https://www.datenschutzerklaerung.de">https://www.datenschutzerklaerung.de</a></p>
```

Comment:
Still a link to the german site?

Reply:
I think this is a link to the generator and is the source of the german and the english text

Reply:
The German text is generated by this site, the English one is translated by deepl